java/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: postgresql
+  repository: https://kubernetes-charts.storage.googleapis.com
+  version: 3.15.0
+digest: sha256:b358f4b5e285cada41065f93c12a687d7c8c25870d036af45cf0a11514aff595
+generated: 2019-03-27T16:47:41.963285Z
```

Comment:
You don't need to check in this file as it's generated automatically.

Reply:
that is fine. Thanks